packages/core/src/shader/utils/logProgramError.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * will log a shader error highlighting the lines with the error
+ * also will add numbers along the side.
+ *
+ * @param gl - the WebGLContext
+ * @param shader - the shader to log errors for
+ */
+function logPrettyShaderError(gl:WebGLRenderingContext, shader:WebGLShader):void
```

Comment:
Recommend wrapping the contents of this function in the following:
```js
// #if _DEBUG

// #endif
```
This would exclude all this work in the min version. maybe consider a simpler version for min version:

```js
// #if _DEBUG
// ... debugging stuff goes here
// #else
console.error('Shader program error');
// #endif
```

Reply:
Yo! I think this is important to have even in the min version - as it's critical when working on shaders of any kind in pixi not just the internal ones iyswim?

Reply:
Okay